src/crypto/slow-hash.c

```diff
@@ -214,6 +215,33 @@ extern void aesb_pseudo_round(const uint8_t *in, uint8_t *out, const uint8_t *ex
     lo ^= SWAP64LE(*(U64(hp_state + (j ^ 0x20)) + 1)); \
   } while (0)
 
+#define VARIANT4_RANDOM_MATH_INIT() \
+  v4_reg r[8]; \
+  struct V4_Instruction code[TOTAL_LATENCY * ALU_COUNT + 1]; \
+  do if (variant >= 4) \
+  { \
+    v4_reg* data = (v4_reg*)(state.hs.w + 12); \
```

Comment:
`memcpy` here for portability. GCC/Clang will generate identical ASM to this invalid aliasing version iff the platform supports mis-aligned moves. Or is this somehow guaranteed alignment?